### PR TITLE
Alternator: Engine Temperature: Added support

### PR DESCRIPTION
**Pull request by nmbath**
Added support to display engine temperature.

### CODE REVIEW

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml [hunk 1]
```diff
@@ -85,6 +85,13 @@ ObjectModel {
 		allowed: defaultAllowed && dataItem.isValid
 	}
 
+	ListTemperatureItem {
+		text: CommonWords.temperature
+		dataItem.uid: root.bindPrefix + "/Engine/Temperature"
+		allowed: defaultAllowed && dataItem.isValid
+		onHeightChanged: console.trace()
```
Comment (chriadam) [inline pages/settings/devicelist/dc-in/PageAlternatorModel.qml:92]: leftover debugging?

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml [hunk 2]
```diff
@@ -85,6 +85,13 @@ ObjectModel {
 		allowed: defaultAllowed && dataItem.isValid
 	}
 
+	ListTemperatureItem {
+		text: CommonWords.temperature
```
Comment (DanielMcInnes) [inline pages/settings/devicelist/dc-in/PageAlternatorModel.qml:89]: The change looks good, but I don't see '.../Engine/Temperature' on the bus (I do see '.../Engine/Speed'). Does this need other customized firmware to work?

Comment (nmbath) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml:89]: Yes this is a new dbus path that has been added through the ve.can driver [here](https://github.com/victronenergy/dbus-vecan/commit/9cb67d8121041330f2257d34b6e84a38bd33b2d2)

**Review by DanielMcInnes — COMMENTED** [review]
lgtm (once the leftover debugging is removed)

**Comment by nmbath** [comment]
I have updated to remove the console.trace().

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml (hunk 2 above)
Comment (jpetrell) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml:88]: Can this be merged? Has the ve.can driver change been included in the existing releases?

Comment (nmbath) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml:88]: is in the main branch of vecan-dbus, however that has not yet been merged into VenusOS beta. It is on the todo list.

**Review by DanielMcInnes — APPROVED** [review]

**Comment by msigman** [comment]
I love the idea of displaying the temp on the alternator box. However one suggestion is that alternator temp would likely be more useful than the engine temp for this. Devices like Wakespeed and Zeus have a temperature probe and already make the alternator temp available in the Cerbo.

**Comment by nmbath** [comment]
Alternator temp is already displayed, as long as it has been supplied by the controller.

**Comment by msigman** [comment]
> Alternator temp is already displayed, as long as it has been supplied by the controller.
> 
> In all cases this is displayed on the details page.

Understood, makes sense. I was thinking this was adding the temp display on the Overview tile, similar to the battery temp display.  Now I understand you are adding it in the Details page for the alternator. Great addition! 

A separate suggestion then for consideration would be to add the alternator temp to the Overview tile, like the battery temp. I suppose this should be logged as a new issue/feature request.
IMG_1163

**Comment by chriadam** [comment]
@nmbath please ping me or Daniel when the required version of vecan-dbus has made its way into venus, so we can merge this one.  Thanks!

**Review by blammit — CHANGES_REQUESTED** [review]
Needs an update to the temperature names, as commented.

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml (hunk 2 above)
Comment (nmbath) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml]: @blammit I have differentiated using` sTrId("dc_alternator_temperature")` and `sTrId("alternator_engine_temperature")` however not sure what else is needed to define these strings or where they are defined.

Comment (blammit) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml]: @nmbath it just needs the Engineering English strings, like this:

```
//% "Alternator temperature"
text:  qsTrId("dc_alternator_temperature")
```

and

```
//% "Engine temperature"
text: qsTrId("alternator_engine_temperature")
```

(If those are the strings that should be displayed.)

Comment (nmbath) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml]: @blammit Many thanks. Done and pushed an udpate.

**Review by blammit — APPROVED** [review]

**Pull request merged**